client/my-sites/current-site/index.jsx

```diff
@@ -113,7 +113,6 @@ const CurrentSite = React.createClass( {
 	},
 
 	previewSite: function( event ) {
-		analytics.ga.recordEvent( 'Sidebar', 'Clicked View Site' );
```

Comment:
Thanks for the overview! 

1. GA change then seems good to me `onPreviewSite`. Do you think would be worth to have also a Tracks event?
2. Right! Page views are more effective at that than sidebar clicks.
3. That tracker is there to test the selling properties, so I'd keep it there. 

I'm thinking it might make sense to have a Tracks property for the purpose of UI flows analysis, something like `calypso_sidebar_mysites` with as property the button. We can add this in a separate PR tho as it's a change on the entire sidebar.